packages/core/admin/admin/src/content-manager/hooks/useFetchContentTypeLayout/index.js

```diff
@@ -68,10 +68,10 @@ const useFetchContentTypeLayout = contentTypeUID => {
   }, [contentTypeUID, getData]);
 
   const updateLayout = useCallback(
-    newLayout => {
+    data => {
       dispatch({
         type: 'UPDATE_LAYOUT',
-        newLayout: formatLayouts({ contentType: newLayout, components: {} }, schemas),
```

Comment:
I'm wondering if the problem could be solved here by reusing the previously received components info as they won't have changed 🤔

Reply:
Well, yes I wanted to do that at first. 
But I allocate myself only a very limited time to look into issues found in external software as I have a massive backlog of things I need to do in my own code base. 

**I could not figure out where the components data is stored on the frontend for reuse** so I figure I could just change 
 the endpoint to return the same object as the GET does. It aligned with the practice we have been following in my work places throughout my career where the PUT to an endpoint will return the modified resource.
In this case GET /configuration returns full configuration, so I made PUT /configuration return full configuration too. 
Yes, it is less efficient in terms of network use but it was quick and straightforward.